std/testing/group.ts

```diff
@@ -0,0 +1,25 @@
+interface DenoTest {
+  (definition: Deno.TestDefinition): void;
+  (title: string, body: Deno.TestFunction): void;
+  (body: Deno.TestFunction): void;
+}
+
+/** Add label as prefix for output text of Deno.test  */
+export function testGroup(label: string): DenoTest {
```

Comment:
Maybe this is something we should do in cli? cc @bartlomieju

Reply:
If there should be discussion, shall I separate changes into another PR?

Reply:
Yes please. That would make it faster to land this.

Reply:
Do we have that discussion already?  What do you mean: `do in cli`?

Reply:
If the only purpose of this method is too provide a prefix to tests then I'm very much against it; just copy-paste prefix several times. Otherwise please revisit #4092 :))